__tests__/app.test.js

```diff
@@ -348,11 +348,92 @@ describe("app", () => {
                 .get("/api/reviews/100/comments")
                 .expect(404)
                 .then(({ body: { message } }) => {
-                  expect(message).toBe("No review found with review id: 100");
+                  expect(message).toBe("Review not found");
                 });
             });
           });
         });
+        describe("/comments", () => {
+          describe("PATCH: /api/reviews/:review_id/comments", () => {
+            test("201: responds with comment object that has been added to database", () => {
+              const commentToPost = {
+                username: "mallionaire",
+                body: "I love this game!",
+              };
+              return request(app)
+                .post("/api/reviews/1/comments")
+                .send(commentToPost)
+                .expect(201)
+                .then(({ body: { comment } }) => {
+                  expect(comment).toEqual({
+                    comment_id: 7,
+                    author: "mallionaire",
+                    body: "I love this game!",
+                    votes: 0,
+                    review_id: 1,
+                    created_at: expect.any(String),
+                  });
+                });
+            });
```

Comment:
You could additionally write a test to check that additional keys on the request object are ignored. Eg if we have a key of votes this is ignored, we get a 201 and the votes is still set to 0.